packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -251,22 +272,54 @@ export default function LabeledField(props: Props) {
         return React.cloneElement(field, {
             id: fieldId,
             "aria-describedby": [
-                errorMessage && errorId,
                 description && descriptionId,
+                readOnlyMessage && readOnlyMessageId,
+                errorMessage && errorId,
```

Comment:
I made a small update to the order of the aria-describedby ids so that the error message is last (similar to the visual order)